test/models.test.js

```diff
@@ -94,6 +94,15 @@ describe('Model tests', function() {
     });
   });
 
+  describe('Model methods', function() {
+    it('should support promises', function(done) {
+      assert(User.create() instanceof Promise);
+      assert(User.find() instanceof Promise);
+      assert(User.findById(99) instanceof Promise);
```

Comment:
FWIW, I think this may not work with loopback@3 where don't use the global `Promise` implementation. A better check would be something like this:

```js
var retval = User.findById();
assert(retval && typeof retval.then === 'function');
```